lib/testing/src/lib/process-services-cloud/pages/start-process-cloud-component.page.ts

```diff
@@ -114,4 +114,12 @@ export class StartProcessCloudPage {
             }
         });
     }
+
+    selectProcessDefinitionAtIndex(index: number) {
+        this.processDefinition.click();
```

Comment:
can you use the common browser action in the  /lib/core/utils/browser-actions.ts to sendKeys?